create.py

```diff
@@ -80,4 +84,62 @@ def logout():
         return render_template("reg.html", var1 = var1)
     except:
         var1 = "You must first log in to logout"
-        return render_template("reg.html",var1 = var1)
\ No newline at end of file
+        return render_template("reg.html",var1 = var1)
+@app.route('/books/<id>',methods=['POST','GET'])
+def books(id):
+    print(id)
+    result = db.session.query(Books).filter(Books.isbn == id).first()
+    print(result)
+    data=Review.query.all()
+    r=Review.query.filter_by(isbn=id).all()
+    print(r)
+    if request.method=='POST':
+        reviewdata=Review(request.form['names'],id,request.form['email'],request.form['comment'],request.form['rating'])
```

Comment:
data must not be again asked to re enter. Use the session variable to fetch the details of the user.

Reply:
updated based on feedback @ashishk144